htdocs/core/modules/facture/doc/pdf_sponge.modules.php

```diff
@@ -1640,7 +1640,7 @@ protected function drawTotalTable(&$pdf, $object, $deja_regle, $posy, $outputlan
 		// Total remise
 		$total_line_remise = 0;
 		foreach ($object->lines as $i => $line) {
-			$total_line_remise += pdfGetLineTotalDiscountAmount($object, $i, $outputlangs, 2); // TODO: add this method to core/lib/pdf.lib
+			$total_line_remise += floatval(pdfGetLineTotalDiscountAmount($object, $i, $outputlangs, 2)); // TODO: add this method to core/lib/pdf.lib
```

Comment:
The doc of pdfGetLineTotalDiscountAmount says the method already return float.
So this should be useless. May be the fix must be done into pdfGetLineTotalDiscountAmount to fix the case it does not return an int or float.

Reply:
it may return also string
![image](https://github.com/Dolibarr/dolibarr/assets/3624836/33df7efa-b03b-4b3f-8493-928e91a4a887)